providers/implementations/encode_decode/decode_der2key.c

```diff
@@ -367,24 +391,33 @@ static int der2key_export_object(void *vctx,
     return 0;
 }
 
+#define D2I_PUBKEY_NOCTX(n, t, f)                                       \
```

Comment:
I can't see that `t` is used anywhere in the generated function...

Reply:
It was originally used to cast the key to the right type, but then it became clear the key is (and MUST be) always NULL, so I replaced the key with `NULL`, which does not need a cast, but the `t` is still there.

I guess I should drop it?

Reply:
Done.